Student-T-Test/Student's t-test.ipynb

```diff
@@ -0,0 +1,869 @@
+      "Requirement already satisfied: mistune>=0.8.1 in /Users/abdullah/anaconda3/lib/python3.7/site-packages (from nbconvert->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (0.8.4)\n",
+      "Requirement already satisfied: pandocfilters>=1.4.1 in /Users/abdullah/anaconda3/lib/python3.7/site-packages (from nbconvert->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (1.4.2)\n",
+      "Requirement already satisfied: defusedxml in /Users/abdullah/anaconda3/lib/python3.7/site-packages (from nbconvert->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (0.6.0)\n",
+      "Requirement already satisfied: entrypoints>=0.2.2 in /Users/abdullah/anaconda3/lib/python3.7/site-packages (from nbconvert->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (0.3)\n",
+      "Requirement already satisfied: testpath in /Users/abdullah/anaconda3/lib/python3.7/site-packages (from nbconvert->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (0.4.2)\n",
+      "Requirement already satisfied: MarkupSafe>=0.23 in /Users/abdullah/anaconda3/lib/python3.7/site-packages (from jinja2->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (1.1.1)\n",
+      "Requirement already satisfied: webencodings in /Users/abdullah/anaconda3/lib/python3.7/site-packages (from bleach->nbconvert->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (0.5.1)\n",
+      "Enabling notebook extension jupyter-js-widgets/extension...\n",
+      "      - Validating: \u001b[32mOK\u001b[0m\n",
+      "Requirement already satisfied: chart_studio in /Users/abdullah/anaconda3/lib/python3.7/site-packages (1.1.0)\n",
+      "Requirement already satisfied: plotly in /Users/abdullah/anaconda3/lib/python3.7/site-packages (from chart_studio) (4.3.0)\n",
+      "Requirement already satisfied: requests in /Users/abdullah/anaconda3/lib/python3.7/site-packages (from chart_studio) (2.22.0)\n",
+      "Requirement already satisfied: retrying>=1.3.3 in /Users/abdullah/anaconda3/lib/python3.7/site-packages (from chart_studio) (1.3.3)\n",
+      "Requirement already satisfied: six in /Users/abdullah/anaconda3/lib/python3.7/site-packages (from chart_studio) (1.12.0)\n",
+      "Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /Users/abdullah/anaconda3/lib/python3.7/site-packages (from requests->chart_studio) (1.24.2)\n",
+      "Requirement already satisfied: idna<2.9,>=2.5 in /Users/abdullah/anaconda3/lib/python3.7/site-packages (from requests->chart_studio) (2.8)\n",
+      "Requirement already satisfied: chardet<3.1.0,>=3.0.2 in /Users/abdullah/anaconda3/lib/python3.7/site-packages (from requests->chart_studio) (3.0.4)\n",
+      "Requirement already satisfied: certifi>=2017.4.17 in /Users/abdullah/anaconda3/lib/python3.7/site-packages (from requests->chart_studio) (2019.9.11)\n"
+     ]
+    }
+   ],
+   "source": [
+    "! pip install ipywidgets\n",
```

Comment:
Split install and import to two different cells for convenient repeated  launch of the notebook.